dev.neovide.neovide.metainfo.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.neovide.neovide</id>
+  <name>neovide</name>
+  <summary>No Nonsense Neovim Client in Rust</summary>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <developer id="dev.neovide">
+    <name>Kaylee Simmons</name>
+  </developer>
+  <description>
+    <p>This is a simple, no-nonsense, cross-platform graphical user interface for Neovim (an aggressively refactored and updated Vim editor). Where possible there are some graphical improvements, but functionally it should act like the terminal UI.</p>
+  </description>
+  <launchable type="desktop-id">dev.neovide.neovide.desktop</launchable>
+  <branding>
+    <color scheme_preference="light" type="primary">#79b242</color>
+    <color scheme_preference="dark" type="primary">#79b242</color>
```

Comment:
Same colour for both, doesn't seem right? https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#good-brand-colors